media/js/externalpages/pocket/mobile-nav.es6.js

```diff
@@ -21,6 +21,7 @@ function detectClickOutside(event) {
 
 function handleMenuOpen() {
     mobileNavWrapper.classList.add('active');
+    mobileNav.ariaExpanded = 'true';
```

Comment:
I tested this out in the inspector tools I don't see the `aria-expanded` attribute on the `mobile-nav` element, so when I open the nav, it still doesn't appear, rendering it to not show up at all.

Reply:
I think this should be:
```suggestion
    mobileNav.setAttribute('aria-expanded, 'true');
```